views/refresh.html

```diff
@@ -0,0 +1,53 @@
+{% set locale = localeInfo.lang %}
+{% set localeDir = localeInfo.locale %}
+<!DOCTYPE html>
+<html lang="{{ locale }}" dir="{{ localeInfo.direction }}">
+  <head>
+    <meta charset="utf-8">
+    <title>{{ gettext("errorSomethingWentWrong") }}</title>
+    <link rel="stylesheet" href="/resources/stylesheets/userbar.css">
+    <link rel="stylesheet" href="/homepage/stylesheets/style.css">
+    <link rel="stylesheet" href="/resources/stylesheets/error.css">
+  </head>
+  <body class="error-page">
+
+    <section class="main-cta">
+      <div class="content-wrapper centered-message">
+
+        <img class="spilled-thimble" src="/img/spilled-thimble.png" />
+        <h1>
+          Oh no, something's wrong!
+        </h1>
+        <h2>
+          {{ gettext("errorCantLoadPreview") | safe }}
+        </h2>
+
+        <p class="button-wrapper">
+          <button class="try-again action-button">
+            {{ gettext("reloadButtonLabel") }}
+          </button>
+        </p>
+        <p>
+          {{ gettext("errorKeepSeeingPage") }}
+        </p>
+
+        <div class="contact-icons">
+          <a title="{{ gettext("contactEmail") }}" class="email" href="mailto:thimble@mozillafoundation.org" target="_blank">
+            <span class="icon"></span>
+          </a>
+          <a title="{{ gettext("contactTwitterTitle") }}" class="twitter" href="https://twitter.com/mozlearn" target="_blank">
```

Comment:
Should we update this to your new twitter account?  Also, we might want to file/fix that in other templates.

Reply:
I'll update all of the Twitter accounts in a separate pass, so let's leave as is.